test/Api.Tests/testconfig.json

```diff
@@ -0,0 +1,6 @@
+﻿// http://docs.mytestedasp.net/tutorial/licensing.html#registering-a-license-code
+// Each license is per web application and hence this is according to our test assembly name.
+// Also it will for all the assemblies that starts with the name "Api".
+{
+  "License": "4433-a4DvjmN8PMZSeaJZs7pw/MQnBFLfu1XGG9PXem39ml5ObzGSdfww7joBm1ADPCxSqc61qK+Zk9dmm9k7vJMHxCe4ASG64LlxpfCkzqGz7XAE0nj/mqz5pTim3pf1OESUo+wGFbj9RvJhlln5kL7hNGz49beoA5ks4hJrLQzNrho0NDMzOjIwMTctMTItMDY6bmFnYXJhakBjYW1wdXMudW5pLXBhZGVyYm9ybi5kZTpVbml2ZXJzaXR5IG9mIFBhZGVyYm9ybiwgR2VybWFueTpBcHBsaWNhdGlvbjpBcGku"
```

Comment:
This is the reply I got from the Admin of My Tested ASP.NET 
[ Ivaylo Kenov, admin@mytestedasp.net ] :

Educational projects receive a license code free of charge.

Each license is per web application, so I will need to know your test assembly name(s) in order to generate you one.

For example, if your test assembly is called “MyWebApp.Test”, the license key will work on every assembly starting with “MyWebApp.”. So if you decide to separate your tests into multiple projects, for example, “MyWebApp.Controllers.Test” and “MyWebApp.ViewComponents.Test”, the license key will still work. 

If you have multiple applications, let me know, and I will give you a license for each one of them. The license is perpetual, and it will work with all library versions released during the next year. Afterward, you can stay with the latest unlocked version or renew the license key.

Once I send you the license code, you can install it by following the steps in the official tutorial available here: http://docs.mytestedasp.net/tutorial/licensing.html#registering-a-license-code

Let me know, if you have further questions.

Kind regards,
Ivaylo
My Tested ASP.NET

The next mail:

this is the license code you need:

4433-a4DvjmN8PMZSeaJZs7pw/MQnBFLfu1XGG9PXem39ml5ObzGSdfww7joBm1ADPCxSqc61qK+Zk9dmm9k7vJMHxCe4ASG64LlxpfCkzqGz7XAE0nj/mqz5pTim3pf1OESUo+wGFbj9RvJhlln5kL7hNGz49beoA5ks4hJrLQzNrho0NDMzOjIwMTctMTItMDY6bmFnYXJhakBjYW1wdXMudW5pLXBhZGVyYm9ybi5kZTpVbml2ZXJzaXR5IG9mIFBhZGVyYm9ybiwgR2VybWFueTpBcHBsaWNhdGlvbjpBcGku

Follow these steps to install it: http://docs.mytestedasp.net/tutorial/licensing.html#registering-a-license-code

As for the build server - you do not have to do anything specific. The license is part of project compilation as you can read from the link above.

Let me know if you need further assistance.

Kind regards,
Ivaylo
My Tested ASP.NET